src/wp-admin/includes/schema.php

```diff
@@ -1058,6 +1058,15 @@ function populate_network( $network_id = 1, $domain = '', $email = '', $site_nam
 		update_user_meta( $site_user->ID, 'source_domain', $domain );
 		update_user_meta( $site_user->ID, 'primary_blog', $current_site->blog_id );
 
+		$wpdb->insert(
```

Comment:
Why is this using `wpdb` directly rather than the options API?

Reply:
Good point. This totally could use `update_network_option()`. 👍

Reply:
Sadly we can not use `update_network_option` in this context. As you can see, line 1042, we are not in a multisite context yet. So if we use `update_network_option` it will fallback update_option. Which is new use.

Reply:
Thanks Jonny, that makes sense.